tools/clang/test/SemaHLSL/template-diag-deferred.hlsl

```diff
@@ -1,3 +1,5 @@
+// RUN: %dxc -Tlib_6_6 -HV 2021 -verify %s
```

Comment:
Is lib_6_6 the lowest SM this test passes in?

Reply:
Yes. Because it used ResourceDescriptorHeap which requires shader model 6.6.